next/src/components/cards/CategoryCard.tsx

```diff
@@ -11,12 +11,16 @@ type Props = {
 } & CardBaseProps
 
 const CategoryCard = ({ title, linkProps, ...rest }: Props) => {
+  const titleId = useId()
+
   return (
     <CardBase className="h-[240px]" {...rest}>
       <CardContent className="h-full justify-between p-6">
         {/* FIXME Typography. Convert to use Typography. Issue: Probably safe to convert but cant find page where is this used for testing */}
-        <h3 className="line-clamp-4 text-h4 group-hover:underline">{title}</h3>
-        <Button variant="link" stretched {...linkProps} />
+        <h3 id={titleId} className="line-clamp-4 text-h4 group-hover:underline">
+          {title}
+        </h3>
```

Comment:
Could you please change this to `Typography variant="h4" as="h3"` (please use no `line-clamp`)? When done, please remove also the comment.